Jenkinsfile

```diff
@@ -34,9 +34,19 @@ if (needSplittingFromWorkspace) {
 
 branches['CI'] = {
   stage('CI') {
-    node('maven-11') {
-      checkout scm 
-      sh "mvn verify --no-transfer-progress -DskipTests -P jenkins-release"
+    retry(count: 2, conditions: [kubernetesAgent(handleNonKubernetes: true), nonresumable()]) {
+      node('maven-11') {
+        checkout scm
+        def mavenOptions = [
+          '-Dset.changelist',
+          '-DskipTests',
+          '-U',
+          'clean',
+          'install',
+        ]
+        infra.runMaven(mavenOptions, 11)
```

Comment:
Internally respects `MAVEN_SETTINGS`, preparing for the potential use of the artifact caching proxy.